### DISCUSSION TITLE
Allow Access to DJs

### DISCUSSION THREAD

**Discussion by TwixtedChaox** [AzuraCast/AzuraCast#6371] [Ideas]
I think we should have a feature with permissions control that allows a DJ to access the administration panel and have access to certain features

some features would be playlists etc.

main reason for this is if a streamer is sick they may want to schedule their show ahead of time using the playlists feature to do so... etc without having to actually connect a live stream.

**Comment by robz0** [comment]
@TwixtedChaox - Have you looked at https://docs.azuracast.com/en/administration/roles-and-permissions ?
Maybe you have and it doesn't do what you want. 
But you can be really flexible with that.
